### DISCUSSION TITLE
The release version of monogame Android does not start

### DISCUSSION THREAD

**Discussion by izjoga1223** [MonoGame/MonoGame#8513] [Android]
In the debug version, the object is initialized and the text is displayed.But in the release version, initialization causes an error, and the build stops.monogame Android does not start

I tried to clean up and rebuild the project in VS.I rebuilt the files in MGCB editor. everything is identical in obj/debug/content and obj/release/content

P.S. I found out that this is happening because of the AOT compilation

No matter what I do, the mistake remains.I really need help!

**Comment by KelsamGames** [comment]
Is it because of this? > https://community.monogame.net/t/trimming-linker-breaks-spritefont-loading/18296/2?u=thekelsam

If so, perhaps this should be added to the Android template's csproj.

**Reply by izjoga1223** [reply]
thank you very much!
